x/feegrant/keeper/keeper.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/feegrant/types"
 )
 
+// FeegrantKeeperI is the interface for x/feegrant's implements.
+type FeegrantKeeperI interface {
```

Comment:
Why are we introducing this interface?

Reply:
We are using expected_keeper interface for feegrant in x/auth/ante .

Reply:
I see. However, we're introducing 2 separate interfaces with the same signature.

In this file, could we remove this interface, and use `var _ ante.FeegrantKeeperI = &Keeper{}`